src/test/java/com/artipie/rpm/hm/NodeHasPkgCountTest.java

```diff
@@ -114,22 +105,13 @@ void doesNotMatchWhenPackageAttributeDiffers() throws FileNotFoundException {
     }
 
     @Test
-    @Disabled
     void describesCorrectlyWhenPackageAttributeDiffers() throws FileNotFoundException {
```

Comment:
@olenagerasimova this test is failing

Reply:
@Vatavuk I can see) It fails on windows only which I do not have, I'll figure it out a bit later